### ISSUE TITLE
Crash report

### ISSUE THREAD

**Issue by ghost** [spoorn/sodium-forge#92]
Frequent teleportation cause this:
https://gist.github.com/ternsip/7df4c04ebe9c0d2764208db6b691539e
+
https://gist.github.com/ternsip/b142e6ae127561909b68fd65b4601be8
+
https://gist.github.com/ternsip/351a1d7b37658a2c46984124065c40bb
+
https://gist.github.com/ternsip/47919b88504ea3aa58d478374cb7b6b6

This prevents me entering the game:
https://gist.github.com/ternsip/e50c7bdb060346f8555db867309dbef2 - removing on client helps.

Also sometimes after teleportation area around becomes dark (no light), reconnecting helps.

**Comment by jclendineng** [comment]
Might want to remove your access token from the gist...unless you are planning on regenning it

**Comment by ghost** [comment]
yeah, thanks, GPU crash-reports does not censor tokens for some reason.

**Comment by spoorn** [comment]
Some of these should be fixed along with https://github.com/spoorn/sodium-forge/issues/91
